doc/changes/latest.inc

```diff
@@ -113,6 +113,8 @@ Enhancements
 Bugs
 ~~~~
 
+- Add Shift_JIST mu in :func:`mne.io.edf.read_raw_edf` (:gh:`10356` by :newcontrib:`Michiru Kaneda`)
```

Comment:
One sticking point that [CircleCI caught](https://app.circleci.com/pipelines/github/mne-tools/mne-python/12967/workflows/90af1314-31ba-4cd0-9a5a-8a3ffc0affd8/jobs/41864) (it treats warnings as errors, so if you search "warning" you'll find it):
```
changes/latest.inc:116: WARNING: py:func reference target not found: mne.io.edf.read_raw_edf
```
We have to refer to these things based on where they're documented on the API page, which in this case is the `mne.io` namespace, not `mne.io.edf`:
```suggestion
- Add Shift_JIST mu in :func:`mne.io.read_raw_edf` (:gh:`10356` by :newcontrib:`Michiru Kaneda`)
```